wikipron/config.py

```diff
@@ -130,7 +130,13 @@ def _get_process_pron(
         def wrapper(pron):
             for processor in processors:
                 pron = processor(pron)
-            return pron
+            # GH-59: Skip prons that are empty, or have only stress marks or
+            # syllable boundaries. The `any()` call is much faster than
+            # re.match(r"[^ˈˌ.]", pron).
```

Comment:
`any` is good for us here as we're only interested in "if there's any" rather than wanting some exact match. For how much faster `any` is compared to `re.match`:

```python
Python 3.7.3 (default, Mar 27 2019, 16:54:48)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.7.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import re

In [2]: %timeit re.match(r"[^ˈˌ.]", "lɪŋ.ˈɡwɪs.tɪks")
718 ns ± 1.78 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [3]: %timeit any(c not in "ˈˌ." for c in "lɪŋ.ˈɡwɪs.tɪks")
321 ns ± 1.8 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```

If I used `re.match` instead of `any`, the smoke test of `wikipron.scrape` in the test suite would fail because of exceeding the timeout limit.

Reply:
what about making those three characters a `frozenset`? Even when the number of things in the set is small I've found that very very fast compared to linear comparison.

Reply:
`frozenset` does seem cleaner here. I'll switch to it. 

It looks like `frozenset` or not doesn't make a lot of difference for performance (I'm on another computer now):

```python
Python 3.6.8 |Anaconda, Inc.| (default, Dec 29 2018, 19:04:46)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.6.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: frozen = frozenset(["ˈ", "ˌ", "."])

In [2]: %timeit any(c not in "ˈˌ." for c in "lɪŋ.ˈɡwɪs.tɪks")
467 ns ± 21.4 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [3]: %timeit any(c not in frozen for c in "lɪŋ.ˈɡwɪs.tɪks")
480 ns ± 32 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [4]: %timeit any(c not in "ˈˌ." for c in "lɪŋ.ˈɡwɪs.tɪks")
457 ns ± 8.82 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [5]: %timeit any(c not in frozen for c in "lɪŋ.ˈɡwɪs.tɪks")
467 ns ± 3.52 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```